cheroot/ssl/builtin.py

```diff
@@ -272,63 +272,20 @@ def bind(self, sock):
 
     def wrap(self, sock):
         """Wrap and return the given socket, plus WSGI environ entries."""
-        EMPTY_RESULT = None, {}
```

Comment:
This should be kept. The variable name is a semantic label of the data it holds.

Reply:
revived.